### PR TITLE
Fix inverted logic setup. Change chip for RPi 5

### PR DESCRIPTION
**Pull request by aguedob**

### CODE REVIEW

File: custom_components/rpi_gpio/__init__.py [hunk 1]
```diff
@@ -44,19 +44,20 @@ def prepare_gpio(event):
...
     global gpiod_config, gpiod_lines
-
+    
     if gpiod_lines:
-        gpiod_lines.__exit__()
+        gpiod_lines.__exit__(None,None,None)
 
     gpiod_lines = gpiod.request_lines(
-        "/dev/gpiochip0",
+        "/dev/gpiochip4",
```
Comment (raboof) [inline custom_components/rpi_gpio/__init__.py]: Can we find a way to autodetect whether 0 or 4 would be the correct default here?

**Review by raboof — COMMENTED** [review]
Generally looks good, but we need a way to select the right `/dev/` device

File: custom_components/rpi_gpio/__init__.py (hunk 1 above)
Comment (aguedob) [reply custom_components/rpi_gpio/__init__.py]: With gpiodetect is possible. Probably libgpiod offers a way for checking it. Let me have a look this afternoon (busy now) and will update my PR

Comment (aguedob) [reply custom_components/rpi_gpio/__init__.py]: Updated my PR. No error handling there, but now it calculates the gpiochip device to use. Tested in Raspberry Pi 3 and 5

File: custom_components/rpi_gpio/__init__.py [hunk 2]
```diff
@@ -42,14 +42,24 @@ def prepare_gpio(event):
     hass.bus.listen_once(EVENT_HOMEASSISTANT_START, prepare_gpio)
     return True
 
+def device_name():
+    for id in range(5):
```
Comment (raboof) [inline custom_components/rpi_gpio/__init__.py:46]: Maybe try more in case the pi6 has more? :)

**Review by raboof — APPROVED** [review]

**Pull request merged**